app/jobs/publish_feed_job.rb

```diff
@@ -7,12 +7,10 @@ class PublishFeedJob < ApplicationJob
 
   attr_accessor :podcast, :episodes, :rss, :put_object, :copy_object
 
-  def perform(podcast)
-    # Since we don't current have a way to retry failed attempts,
-    # and until somthing akin to https://github.com/PRX/feeder.prx.org/issues/714 lands
-    # the RSS publishing is extracted from the publishing pipeline semantics.
-    # TODO: recombine the publishing invocations (RSS, Apple) once we have some retry guarantees
-    podcast.feeds.each { |feed| save_file(podcast, feed) }
```

Comment:
Returns the RSS publishing to it's previous place in the partitioned concurrency semantics. So now Apple publishing and delivery will block RSS, for those podcasts with that configured.